src/containerapp/HISTORY.rst

```diff
@@ -4,13 +4,20 @@ Release History
 ===============
 upcoming
 ++++++
+
+0.3.52
+++++++
+* 'az containerapp env java-component admin-for-spring': Support create/update/show/delete Admin for spring
+* 'az containerapp env java-component nacos': Support create/update/show/delete Nacos
+* 'az containerapp env java-component': Support --bind, --unbind to bind or unbind with other Java Components
```

Comment:
These command are supported from PR: https://github.com/Azure/azure-cli-extensions/pull/7622
It's history added to wrong version `0.3.51`, fixed.